
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1264928390 -3600
# Node ID 17615367bcb7bebe8fd23b7de1802f891b747950
# Parent  9c052867050c10a6b8d05bdb8a1313b5d9a8999d
libv4l: update upside-down device list

From: Hans de Goede <hdegoede@redhat.com>

libv4l: update upside-down device list

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Thu Jan 28 15:58:25 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Sun Jan 31 09:59:50 2010 +0100
@@ -239,8 +239,12 @@
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x090c, 0xe370, 0, "ASUSTeK Computer Inc.        ", "VX3       ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x13d3, 0x5094, 0, "ASUS CORPORATION            ", "K50C      ",
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x13d3, 0x5094, 0, "ASUSTeK Computer Inc.        ", "P50IJ     ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x174f, 0x1408, 0, "ASUSTeK Computer Inc.        ", "K40IN      ",
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x174f, 0x5a35, 0, "ASUSTeK Computer Inc.        ", "F3Ka      ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x174f, 0x5a35, 0, "ASUSTeK Computer Inc.        ", "F3Ke      ",

